tests/simulate/test_hoomd.py

```diff
@@ -0,0 +1,109 @@
+"""Unit tests for relentless.simulate.hoomd."""
+import os
+import tempfile
+import unittest
+
+import gsd.hoomd
+import hoomd
+import numpy as np
+
+import relentless
+
+class test_HOOMD(unittest.TestCase):
+    """Unit tests for relentless.HOOMD"""
+
+    def setUp(self):
+        self._tmp = tempfile.TemporaryDirectory()
+        self.directory = relentless.Directory(self._tmp.name)
+
+    #mock gsd file for testing
+    def create_gsd(self):
+        os.chdir(self.directory.path)
+        f = gsd.hoomd.open(name='test.gsd', mode='wb')
+        s = gsd.hoomd.Snapshot()
+        s.particles.N = 5
+        s.particles.types = ['A','B']
+        s.particles.typeid = [0,0,1,1,1]
+        s.particles.position = np.random.random(size=(5,3))
+        s.configuration.box = [2,2,2,0,0,0]
+        f.append(s)
+        f.close()
+        return f
+
+    def test_initialize(self):
+        """Test running initialization simulation operations."""
+        ens = relentless.Ensemble(T=1.0, V=relentless.Cube(L=2.0), N={'A':2,'B':3})
+        pot = relentless.PairMatrix(types=ens.types)
+        for pair in pot:
+            pot[pair]['r'] = np.array([1.,2.,3.])
+            pot[pair]['u'] = np.array([2.,4.,6.])
+
+        #InitializeFromFile
+        f = self.create_gsd()
+        op = relentless.simulate.hoomd.InitializeFromFile(filename=f.file.name)
+        h = relentless.simulate.hoomd.HOOMD(operations=op)
+        sim = h._new_instance(ens, pot, self.directory)
+        op(sim)
+        self.assertIsInstance(sim._opdata[op].neighbor_list, hoomd.md.nlist.tree)
```

Comment:
These are supposed to accessible like:
```suggestion
        self.assertIsInstance(sim[op].neighbor_list, hoomd.md.nlist.tree)
```